lib/features/family/shared/design/components/content/fun_progressbar.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+import 'package:givt_app/features/family/shared/widgets/texts/label_small_text.dart';
+import 'package:givt_app/features/family/utils/family_app_theme.dart';
+
+class FunProgressbar extends StatefulWidget {
+  const FunProgressbar({
+    required this.currentProgress,
+    required this.total,
+    super.key,
+    this.prefixWidget,
+    this.suffix,
+    this.completedText,
+    this.margin,
+    this.backgroundColor,
+    this.progressColor,
+    this.textColor,
+  })  : assert(currentProgress >= 0, 'Current progress must be non-negative'),
```

Comment:
I didn't know the assert thing :) but it seems like it's only validating that by default on debug versions and not the 'release' version of the app.

Should it be validated there as well? Then we need to add --enable-asserts to the build release command

Reply:
@TammiLion ^^

Reply:
Good point, considering the values will be coming from the BE I don't think that's particularily useful. I will make sure the widget just hides itself rather than gives an error for the release version.